packages/react-resizable-panels-theme/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@salt-ds/react-resizable-panels-theme",
+  "version": "0.0.0",
```

Comment:
Have a try on release a snapshot on version number. Likely need changeset for this new package as well

Reply:
Changeset was added